packages/remix/package.json

```diff
@@ -27,7 +27,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@sentry/cli": "2.2.0",
+    "@sentry/cli": "2.20.4",
```

Comment:
Let's bump this to 2.20.5 instead. It contained a pretty critical performance fix.